test/functional/cmdLineTests/runtimemxbeanTests/playlist.xml

```diff
@@ -23,14 +23,14 @@
 -->
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TestConfig/playlist.xsd">
+	<include>path.mk</include>
```

Comment:
can we make this file name more generic in case we need to add other settings into it?

Reply:
maybe change it to `variables.mk`?

Reply:
sure, that's works. Thanks.